luamin.js

```diff
@@ -171,6 +171,19 @@
 		if (lastCharA == '' || firstCharB == '') {
 			return a + b;
 		}
+		if (separator == ';' && firstCharB == '(') {
+			if (
```

Comment:
Note: This entire if actually ended up being a collection of every valid ending to a lua statement. There are times where this optimsation is possible (E.g. `t={};(a or b)(c)` is 2 statements with or without a `;`), but we can't be 100% sure from text alone it wasn't something like `t=f{};(a or b)(c)` instead (which would change its meaning with no `;`), so we'd need to know the exact statement instead of just the 2 peices of text to be joined, and we just don't provide that right now